.github/workflows/lib-injection.yml

```diff
@@ -115,16 +115,16 @@ jobs:
         run: |
           curl http://localhost:18080
           sleep 1  # wait for traces to be sent
+      - name: Check mocked telemetry
```

Comment:
Is there anything we can print here as confirmation that something actually happened?

Hard to tell from the existing logs: https://github.com/DataDog/dd-trace-py/actions/runs/9389337013/job/25856592783#step:6:3


dumping the telemetry could be good for human audit, wdyt?

Reply:
The validation script prints debug output https://github.com/DataDog/dd-trace-py/actions/runs/9467691845/job/26082304539?pr=9477